controllers/cloud.redhat.com/utils/utils.go

```diff
@@ -234,44 +234,22 @@ func MakePVC(pvc *core.PersistentVolumeClaim, nn types.NamespacedName, labels ma
 	}
 }
 
-// DeploymentStatusChecker takes a deployment and returns True if it is deemed ready by the logic in
-// the function.
-func DeploymentStatusChecker(deployment *apps.Deployment) bool {
-	if deployment.Generation > deployment.Status.ObservedGeneration {
-		// The deployment controller still needs to reconcile
-		return false
-	}
-
-	if deployment.Spec.Replicas != nil {
-		if *deployment.Spec.Replicas == 0 {
-			// Since there's no replicas, there's nothing to do to get ready
-			return true
-		}
-
-		if deployment.Status.UpdatedReplicas < *deployment.Spec.Replicas {
-			// This indicates that the deployment has not completed rolling out the new ReplicaSet
-			return false
-		}
-	}
+// IntPtr returns a pointer to the passed integer.
```

Comment:
These have been moved - can you use the ones in `common.go`?

Reply:
This particular func wasn't moved. Want me to move it as part of this PR?

```
 grep IntPtr -r *
controllers/cloud.redhat.com/providers/kafka/appinterface.go:			Port:     utils.IntPtr(9092),
controllers/cloud.redhat.com/providers/kafka/localkafka.go:			Port:     utils.IntPtr(29092),
controllers/cloud.redhat.com/providers/web/default.go:	c.WebPort = utils.IntPtr(int(web.Env.Spec.Providers.Web.Port))
controllers/cloud.redhat.com/providers/web/default.go:	c.PublicPort = utils.IntPtr(int(web.Env.Spec.Providers.Web.Port))
controllers/cloud.redhat.com/providers/web/default.go:	c.PrivatePort = utils.IntPtr(int(privatePort))
controllers/cloud.redhat.com/utils/utils.go:// IntPtr returns a pointer to the passed integer.
controllers/cloud.redhat.com/utils/utils.go:func IntPtr(i int) *int {
```

Reply:
Can this be a follow up item?